bin/version.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Use of this source code is governed by The MIT License.
+# See the LICENSE file for details.
+
+# Script to udpate the version of SPF and generate docs.
+#
+# Author: nicksay@google.com (Alex Nicksay)
+
+# The script must be passed a version.
+if [[ $# < 1 ]]; then
+  echo "Usage: $(basename $0) [<newversion> | major | minor | patch ]"
+  exit 1
+fi
+
+# Change to the root directory.
+cd "$(dirname $(dirname "$0"))"
```

Comment:
Directory changes inside a subshell don't propagate to the parent shell, so `cd` inside a script doesn't change the current directory